util/chunk/compare.go

```diff
@@ -163,6 +164,11 @@ func cmpJSON(l Row, lCol int, r Row, rCol int) int {
 
 // Compare compares the value with ad.
 func Compare(row Row, colIdx int, ad *types.Datum) int {
+	return CompareOpt(row, colIdx, ad, false)
+}
+
+// CompareOpt compares the value with ad with prefixAsEq optional
+func CompareOpt(row Row, colIdx int, ad *types.Datum, prefixAsEq bool) int {
```

Comment:
Please add comments for `prefixAsEq`.

Reply:
emmm..this param is stranger, I tried to refactor it

Reply:
prefixAsEq